tests/test_obstacle_map.py

```diff
@@ -0,0 +1,107 @@
+from SDA import *
+import unittest
+import numpy 
+
+class Test_Obstacle_map(unittest.TestCase):
+	"""docstring for test_Obstacle_map"""
+	
+	def setUp(self):
+		self.test_drone_point=numpy.array([0,0,0])
+		self.test_fly_zones=numpy.array([numpy.array([(0,0), (0,100), (100,100), (100,0)])])
+		self.obstacle_map=ObstacleMap(self.test_drone_point,self.test_fly_zones)
+
+	def test_add_obstacle(self):
+
+		# test if the last obstacle in obstacle_map is the added object
+		
+		self.test_Obstacle=StationaryObstacle(numpy.array([1,1,1]),2,1)
+		self.obstacle_map.add_obstacle(self.test_Obstacle)
+		
+		self.assertEqual(self.obstacle_map.get_obstacles(),self.test_Obstacle)
```

Comment:
Does this test complete successfully? I do not think this is how the return works...